### PR TITLE
UI Adjustments

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

File: screen/lnd/scanLndInvoice.js [hunk 1]
```diff
@@ -50,7 +48,7 @@ export default class ScanLndInvoice extends React.Component {
     };
   }
 
-  async onBarCodeScanned(ret) {
+  async onBarCodeRead(ret) {
```
Comment (Overtorment) [inline screen/lnd/scanLndInvoice.js]: How did it work before?
Something is not right here...

File: screen/lnd/scanLndInvoice.js [hunk 2]
```diff
@@ -222,8 +220,7 @@ export default class ScanLndInvoice extends React.Component {
                 }}
                 onPress={() => {
                   this.setState({
-                    type:
-                      this.state.type === BarCodeScanner.Constants.Type.back
+                    type: this.state.type === BarCodeScanner.Constants.Type.back
```
Comment (Overtorment) [inline screen/lnd/scanLndInvoice.js]: `npm run lint ` agrees with that?

File: screen/send/scanQrAddress.js [hunk 3]
```diff
@@ -77,8 +77,7 @@ export default class CameraExample extends React.Component {
                 }}
                 onPress={() => {
                   this.setState({
-                    type:
-                      this.state.type === BarCodeScanner.Constants.Type.back
+                    type: this.state.type === BarCodeScanner.Constants.Type.back
```
Comment (Overtorment) [inline screen/send/scanQrAddress.js]: `npm run lint` agrees with that?

File: loc/en.js [hunk 4]
```diff
@@ -87,15 +87,15 @@ module.exports = {
       conf: 'conf',
     },
     details: {
-      title: 'transaction details',
+      title: 'Transaction',
```
Comment (Overtorment) [inline loc/en.js]: was decapetalized because other screens have title decapitalized also

File: loc/en.js [hunk 5]
```diff
@@ -87,15 +87,15 @@ module.exports = {
...
       from: 'Inputs',
       to: 'Outputs',
     },
   },
   send: {
```
Comment (Overtorment) [inline loc/en.js]: send/list & receive/list screens were completely removed in last refactoring commit.
remove it

File: screen/wallets/list.js [hunk 6]
```diff
@@ -1,5 +1,5 @@
 import React, { Component } from 'react';
-import { View, TouchableOpacity, Text, FlatList } from 'react-native';
+import { View, TouchableOpacity, Text, FlatList, RefreshControl } from 'react-native';
```
Comment (Overtorment) [inline screen/wallets/list.js]: is that pull-down to refresh? cool if yes!

File: screen/wallets/list.js [hunk 7]
```diff
@@ -321,161 +312,151 @@ export default class WalletsList extends Component {
...
-                      fontWeight: 'bold',
-                      fontSize: 24,
-                      color: BlueApp.settings.foregroundColor,
-                    }}
-                  >
-                    {loc.transactions.list.title}
-                  </Text>
-                  {(() => {
-                    if (this.state.showRereshButton) {
-                      return <BlueRefreshIcon onPress={() => this.refreshTransactions()} />;
```
Comment (Overtorment) [inline screen/wallets/list.js]: if its removed - remove from BlueComponents

File: screen/wallets/scanQrWif.js [hunk 8]
```diff
@@ -192,8 +192,7 @@ export default class ScanQrWif extends React.Component {
                       }}
                       onPress={() => {
                         this.setState({
-                          type:
-                            this.state.type === BarCodeScanner.Constants.Type.back
+                          type: this.state.type === BarCodeScanner.Constants.Type.back
```
Comment (Overtorment) [inline screen/wallets/scanQrWif.js]: lint

File: screen/lnd/scanLndInvoice.js [hunk 9]
```diff
@@ -19,9 +19,7 @@ const { width } = Dimensions.get('window');
 
 export default class ScanLndInvoice extends React.Component {
   static navigationOptions = {
-    header: ({ navigation }) => {
-      return <View />;
-    },
+    tabBarVisible: false,
```
Comment (Overtorment) [inline screen/lnd/scanLndInvoice.js]: I have a feeling that youre not up-to-date with master...

File: screen/wallets/list.js (hunk 6 above)
Comment (marcosrdz) [reply screen/wallets/list.js]: it is.

File: loc/en.js (hunk 4 above)
Comment (Overtorment) [reply loc/en.js]: still capitalized. not consistent with the rest of the screens

File: loc/en.js [hunk 10]
```diff
@@ -127,10 +124,7 @@ module.exports = {
     },
   },
   receive: {
-    list: {
-      tabBarLabel: 'Receive',
-      header: 'Receive',
-    },
+    header: 'Receive',
```
Comment (Overtorment) [inline loc/en.js]: [22:43:33] TypeError: TypeError: TypeError: TypeError: undefined is not an object (evaluating 'loc.receive.list.header')

when opening Receive screen

**Pull request merged**